### DISCUSSION TITLE
Not starting on Windows - Read-Only Directory Error

### DISCUSSION THREAD

**Discussion by Myself04-88** [diasurgical/DevilutionX#1862] [Q&A] [answered]
Hello,

Devilution is not starting on my PC with following error message:
"Read-Only Directory Error
Unable to write to location:
\Gaétan\AppData\Roaming\diasurgical\devilution"

No matter if the MPQ are copied locally or under appdata.

Any idea ? Maybe it's due to the accent in my user name ?

BR,

**Comment by AJenbo** [comment]
We currently do not support international letters (`é`) in the install path. To workaround this you need to either create a user with out international symbols, or move point the game to a save path outside of your home folder using `--save-dir`.

**Reply by julealgon** [reply]
@AJenbo , perhaps one could use `subst` for this: map a drive letter to the problematic folder, and then redirect the paths to that letter.

This would avoid the issue on Windows while still persisting the data into the same folder.

https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/subst

**Reply by AJenbo** [reply]
Yeah looks like that is an option, but still it's a workaround that has to be deployed by the user. Hopefully we can instead solve the issue and handle this correctly. But currently I do not have the slightest guess where the issue lies.

Here is the task for fixing it https://github.com/diasurgical/devilutionX/issues/1012 if anyone wants to give it a shot

**Reply by julealgon** [reply]
Yeah, I was only suggesting it as a user workaround as well (that still saves to the standard folder). It would definitely be preferred to just support it out of the gate indeed.

**Comment by Myself04-88** [comment, marked as answer]
Ok, issue solved.

I created a shortcut, and in the "target" area I put the following:
D:\devilutionx\devilutionx.exe --save-dir "D:\devilutionx\Save"

And that worked like a charm

Thank you very much ;-)

**Reply by JFBrittes** [reply]
Hello,

I'm having the same problem reported "Read-Only Directory Error Unable to write to location: \Gaétan\AppData\Roaming\diasurgical\devilution"

I feel the issue is the same as my user name also has an international symbol on it.

However, I don't know anything about programming or coding. I just really want to try out this mod.

Can you tell me step by step how to work around this? I don't know how to create the shortcut or what files should go where. :(

**Reply by AJenbo** [reply]
Try the latest beta, the issue should be fixed there:
https://github.com/diasurgical/devilutionX/suites/2729800460/artifacts/60246351

**Comment by JFBrittes** [comment]
Thank you so much it worked!






Em quarta-feira, 12 de maio de 2021 20:14:53 BRT, Anders Jenbo ***@***.***> escreveu: 







Try the latest beta, the issue should be fixed there:
https://github.com/diasurgical/devilutionX/suites/2729800460/artifacts/60246351
